action/output.php

```diff
@@ -47,10 +47,12 @@ public function handleOutput(Doku_Event $event, $param)
         if (!page_exists($ID)) return;
 
         $pos = 0;
+        $ins = -1;
+
         // display struct data at the bottom?
         if ($this->getConf('bottomoutput')) {
             $ins = count($event->data->calls);
-        } else {
+        } else if (!$this->getConf('topoutput')) {
             $ins = -1;
```

Comment:
This line is obsolete now, right?

Reply:
Indeed, thanks! I pushed a fixup for this!